Makefile

```diff
@@ -0,0 +1,12 @@
+all:
+	cargo build
```

Comment:
Consider renaming this make target to `build` because `all` seems confusing.

Reply:
Done [e77c97c](https://github.com/tomip01/conways/pull/4/commits/e77c97c42a4d9a067db2e0db5744b42968d0114b)